helloworld/README.md

```diff
@@ -0,0 +1,33 @@
+# helloworld
+This is a very basic example of how to use
+[Trillian](http://github.com/google/trillian) to implement the operations
+needed for a verifiable log.  It intentionally avoids
+defining any interface between the client and personality, leaving only the
+gRPC interface between the personality and the (Trillian) log.  In a real
+deployment it would be necessary to define an interface between the client
+and the personality, and thus implement a "real" client on top of the
```

Comment:
This sentence is a little unclear to me (from "and thus" onwards). I think it might be better to replace this with something like ", for example a RESTful API or gRPC".